src/apps/inventory/views.py

```diff
@@ -45,8 +45,38 @@ def get_queryset(self):
 
 class ProductListView(ListView):
     model = Product
-    paginate_by = 10
     template_name = "product_list.html"
+    context_object_name = "products"
```

Comment:
take a look at django-filters library

Reply:
I looked through it, not sure that we need it though at least for now, seams like it wont save a lot, and we wont have other things to filter besides maybe categories and i think your advice on not adding libraries is relevant here. And finally i dont think it looks that ugly for now

Reply:
true, it looks good

but django-filters is kinda standard way here. so I personally think you should know how to use it

Reply:
ok, will try to rewrite it with django-filter, but on Monday, thanks for clarification))